ddl/owner_manager.go

```diff
@@ -112,6 +114,17 @@ func (m *ownerManager) SetBgOwner(isOwner bool) {
 // ManagerSessionTTL is the etcd session's TTL in seconds. It's exported for testing.
 var ManagerSessionTTL = 60
 
+// setManagerSessionTTL sets the ManagerSessionTTL value, it's used for testing.
+func setManagerSessionTTL() error {
```

Comment:
I mean set ManagerSessionTTL according to environment variable. If Atoi returns error, we could keep ManagerSessionTTL unchanged and print a warning log.